### DISCUSSION TITLE
Does RootEncoder support RTMPS?

### DISCUSSION THREAD

**Discussion by IamBruce828** [pedroSG94/RootEncoder#1616] [General]
I have tested the code for youtube that runs successful.
But when I use facebook live streaming, it seems the connection will disconnect and retry.
The difference between youtube and facebook is rtmp and rtmps.
 Does RootEncoder natively support RTMPS?

**Comment by pedroSG94** [comment]
Yes, it is supported and that issue should be solved here (apparently it is an error in ktor and I reported the error but for now I did a temporal fix):
https://github.com/pedroSG94/RootEncoder/commit/546ef083a9f704a563e9ec789a7176237db40c29
Are you using the last commit?

**Reply by IamBruce828** [reply]
It works fine when using facebook live streaming by using last commit. Thanks for help.
